requirements.txt

```diff
@@ -32,6 +32,7 @@ openpyxl==3.1.2                 # For Microsoft Excel Files
 svglib==1.5.1                   # Used for creating vouchers from SVG tremplate
 reportlab==3.6.12               # For generating PDFs and graphics
 pyyaml==6.0                     # Used for generating environment variable file
+icalendar==5.0.4                # For iCalendar event export
```

Comment:
We already have the `django-ical` dependency included above. We only want to use one ical implementation.

Reply:
django-ical actually depends on icalendar, so it's fine for me to leave it away in this requirements.txt with no need for me to change my implementation